### PR TITLE
Recipe api testing - WORK WITH CJ 

### PR DESCRIPTION
**Pull request by techprojsforfun**
make rspec tests for api call in recipe model and for the call in userscontroller#show

### CODE REVIEW

**Review by cheolhojang — COMMENTED** [review]
worked on testing with Omid to make the rspec tests pass

File: spec/recipe_spec.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  it 'should be a 200 OK' do
+    expect(Recipe.find_in_api.code).to eq "200"
+  end
+  it 'should return a string' do
+    expect(Recipe.find_in_api.body).to be_an_instance_of(String)
+  end
+  it 'should include "is a test" in body' do
+    expect(Recipe.find_in_api.body).to include('is a test')
+  end
+end
```
Comment (ChloeGuyue) [inline spec/recipe_spec.rb:24]: thorough tests

File: coverage/.resultset.json [hunk 2]
```diff
@@ -1,15 +1,39 @@
 {
   "RSpec": {
     "coverage": {
-      "/Users/omidafshar/Documents/CS169/Grocer-Plus/spec/hi_spec.rb": [
+      "/Users/babajoon/nowtmp/Grocer-Plus/spec/recipe_spec.rb": [
```
Comment (ChloeGuyue) [inline coverage/.resultset.json:4]: These are files from personal machines. This should be added to a .gitignore

Comment (techprojsforfun) [reply coverage/.resultset.json:4]: Ok, will do!

**Pull request merged**